build/MSBuildExtensions.targets

```diff
@@ -1,12 +1,54 @@
 <Project>
   <Target Name="GenerateMSBuildExtensions"
-          DependsOnTargets="GenerateBundledVersionsProps">
+          DependsOnTargets="GenerateBundledVersionsProps;RestoreMSBuildExtensionsPackages">
 
     <ItemGroup>
       <MSBuildExtensionsContent Include="$(GeneratedMSBuildExtensionsDirectory)/**/*" />
+
+      <MSBuildExtensionsContent Include="$(MSBuildExtensionsNuPkgPath)\msbuildExtensions\**\*.*"
+                                Exclude="$(MSBuildExtensionsNuPkgPath)\msbuildExtensions\Microsoft.NET.Build.Extensions\net*\**" />
+      <MSBuildExtensionsContent Include="$(MSBuildExtensionsNuPkgPath)\msbuildExtensions-ver\**\*.*"
+                                DeploymentSubpath="$(MSBuildExtensionsVersionSubfolder)/" />
+      <MSBuildExtensionsContent Include="$(NETStandardLibraryNETFrameworkNuPkgPath)\build\**\*.*"
```

Comment:
Can you please add a comment here that indicates we are ignoring the Framework-specific content from the msbuild extensions package so that we don't need to flow NETStandard.Library.NETFramework through the SDK repo?